src/xpk/commands/cluster.py

```diff
@@ -506,6 +506,16 @@ def run_gke_cluster_create_command(
     addons_str = ','.join(addons)
     command += f' --addons={addons_str}'
 
+  if args.enable_gcpfilestore_csi_driver:
```

Comment:
I don't understand why it is doubles. I fixed this changes(same as print(yaml_string)) in my PR. Please clean this up or change to draft

Reply:
Right, I was careful with the merge, but haven't noticed duplication, sorry.